qubesadmin/tools/qvm_start_daemon.py

```diff
@@ -351,28 +390,42 @@ def on_domain_start(self, vm, _event, **kwargs):
         except qubesadmin.exc.QubesException as e:
             vm.log.warning('Failed to start GUI for %s: %s', vm.name, str(e))
 
+        try:
+            if getattr(vm, 'audiovm', None) != vm.app.local_name:
+                return
+            if not vm.features.check_with_template('audio', True):
+                return
+            if kwargs.get('start_audio', 'True') == 'True':
+                asyncio.ensure_future(self.start_audio_for_vm(vm))
```

Comment:
This part (besides kwargs check) looks like a duplicate of `start_audio` function. Any reason to not simply call it here?

Reply:
It was for keeping same structure/hierarchy than `gui` part.